day_6/exercises/burrito.rb

```diff
@@ -11,9 +11,31 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping (topping)
+    @toppings.push(topping)
+    puts "Burrito toppings are now: #{toppings}"
+  end
+
+  def remove_topping(topping)
+    bad_topping = @toppings.find_index(topping)
+    @toppings.delete_at(bad_topping)
+    puts "Burrito toppings are now: #{toppings}"
+  end
+
+  def change_protein(new_protein)
+    @protein = new_protein
+  end
 end
 
 dinner = Burrito.new("Beans", "Rice", ["cheese", "salsa", "guacamole"])
 p dinner.protein
 p dinner.base
 p dinner.toppings
+puts "What topping would you like to add?"
```

Comment:
Love this interaction pattern here and below - also love the puts within your methods to act as sanity checks!